src/com/facebook/buck/ide/intellij/IjProjectBuckConfig.java

```diff
@@ -121,6 +121,7 @@ public static IjProjectConfig create(
         .setGeneratingAndroidManifestEnabled(
             buckConfig.getBooleanValue(
                 INTELLIJ_BUCK_CONFIG_SECTION, "generate_android_manifest", false))
+        .setOutputUrl(buckConfig.getValue(INTELLIJ_BUCK_CONFIG_SECTION, "output_url"))
```

Comment:
Can you rename this option to be something like `project_compiler_output_url` instead? Right now it's too generic.